apidoc/ostree-sections.txt

```diff
@@ -246,6 +246,7 @@ ostree_repo_remote_get_gpg_verify
 ostree_repo_remote_get_gpg_verify_summary
 ostree_repo_remote_gpg_import
 ostree_repo_remote_fetch_summary
+ostree_repo_remote_fetch_summary_with_options
```

Comment:
I didn't run it before, but yeah, it actually failed. Sorry about that.

Updated the file, hopefully I did that right - tests are passing for me now.